src/_settings.scss

```diff
@@ -474,3 +475,13 @@ $size-topbar-user-toggle-icon: 24px !default;
 $w-topbar-user-nav: 90% !default;
 $maw-topbar-user-nav: 350px !default;
 $bgc-topbar-user-cta: $c-accent !default;
+
+// Thumbnail
+$bgc-thumbnail: $c-white !default;
+$bd-thumbnail: 1px solid $c-gray-lightest !default;
+$p-thumbnail: ceil($p-base / 6) !default;
```

Comment:
just trying to avoid to has different padding definitions.
keep in mind that since you are defining a sui component, it will be linked to theme basic definition. then, every vertical will redefine this variables. 
like this third option. anyways, we need a sui margins definition.